### DISCUSSION TITLE
Youtube livestream issues: no resolution upscaling for flat projection, 360 projection not working.

### DISCUSSION THREAD

**Discussion by wakass** [mozilla/hubs#3436] [General Questions & Discussion] [answered]
Hey all, was wondering if anyone has had any experience in embedding youtube livestreams in hubs. Running into a few issues myself for a supercool small festival with a livestream participation in a hubs hangout! Mostly it seems that the video isn't upscaling to higher resolutions at all, and 360 video seems broken completely? (Private server) Was hoping anyone would have pointers to what I could fix!

**Comment by pattersonbl2** [comment, marked as answer]
We haven't ran into any issues lately with youtube live streams. if you are still needing help on this issue please unmark as answered.
